client/state/ui/actions.js

```diff
@@ -35,9 +35,13 @@ export function setAllSitesSelected() {
 
 export function setSection( section, options = {} ) {
 	options.type = SET_SECTION;
+
 	if ( section ) {
 		options.section = section;
 	}
+
+	// Is this even used now? https://github.com/Automattic/wp-calypso/blob/master/client/layout/index.jsx#L146
```

Comment:
`ag`ing for `hasSidebar` still gives a couple of occurrences. Getting rid of them would be beyond the scope of this PR, but you could file an issue.